packages/mobile/src/components/premium-content-purchase-drawer/PremiumContentPurchaseDrawer.tsx

```diff
@@ -384,7 +381,7 @@ const RenderForm = ({
                     isExistingBalanceDisabled={isExistingBalanceDisabled}
                     showExistingBalance={!!(balance && !balance.isZero())}
                     isCoinflowEnabled={showCoinflow}
-                    isPayWithAnythingEnabled={isPayWithAnythingEnabledFlag}
```

Comment:
nit: should probably remove this prop

Reply:
so there is another usage of payment-method that relies on that being false, surprisingly